tests/DatabaseAdapterTest.php

```diff
@@ -159,6 +159,27 @@ public function testUpdatePolicies()
         ], Enforcer::getPolicy());
     }
 
+    public function testUpdateFilteredPolicies()
+    {
+        // $this->assertEquals([
+        //     ['alice', 'data1', 'read'],
+        //     ['bob', 'data2', 'write'],
+        //     ['data2_admin', 'data2', 'read'],
+        //     ['data2_admin', 'data2', 'write'],
+        // ], Enforcer::getPolicy());
+
+        // Enforcer::updateFilteredPolicies([["alice", "data1", "write"]], 0, "alice", "data1", "read");
+        // Enforcer::updateFilteredPolicies([["bob", "data2", "read"]], 0, "bob", "data2", "write");
+
+        // $policies = [
+        //     ['alice', 'data1', 'write'],
+        //     ['bob', 'data2', 'read'],
+        //     ['data2_admin', 'data2', 'read'],
+        //     ['data2_admin', 'data2', 'write']
+        // ];
+        // $this->assertEqualsCanonicalizing(Enforcer::getPolicy(), $policies);
+    }
+

```

Comment:
Why is it commented out this code? @basakest

Reply:
This method calls some methods in php-casbin, and the pr has not been merged before @techoner